File: VB/CustomFilterPopupExample/CustomGridFilterColumn/CustomTableView.vb
Imports Microsoft.VisualBasic
Imports DevExpress.Xpf.Grid
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace CustomFilterPopupExample
	Friend Class CustomTableView
		Inherits TableView
		Protected Overrides Function CreateFilterColumn(ByVal column As ColumnBase, ByVal useDomainDataSourceRestrictions As Boolean, ByVal useWcfSource As Boolean) As GridFilterColumn
			Return New CustomGridFilterColumn(column, useDomainDataSourceRestrictions, useWcfSource)
		End Function
	End Class
End Namespace

File: CS/CustomFilterPopupExample/CustomGridFilterColumn/CustomTableView.cs
using DevExpress.Xpf.Grid;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CustomFilterPopupExample
{
    class CustomTableView : TableView
    {
        protected override GridFilterColumn CreateFilterColumn(ColumnBase column, bool useDomainDataSourceRestrictions, bool useWcfSource)
        {
            return new CustomGridFilterColumn(column, useDomainDataSourceRestrictions, useWcfSource);
        }
    }
}
